VultisigApp/VultisigApp/Views/Vault/RegisterVaultView.swift

```diff
@@ -67,7 +67,7 @@ struct RegisterVaultView: View {
     }
     
     var label: some View {
-        FilledButton(title: "complete")
+        FilledButton(title: "Save Vault QR")
```

Comment:
You need to create an item in `Localizable.strings` , and use the name here , so the text is translated into other languages

Reply:
Updated the localizable files for all the languages.